src/App.tsx

```diff
@@ -51,7 +50,7 @@ const App: React.FC = () => {
   return (
     <Container>
       <Header />
-      {networkMap.has(safe.chainId) ? (
```

Comment:
Here we should add a warning or a small warning banner on top of the app. 
We could also separate this into an issue and I add it in January.

Reply:
I would prefer to keep that separate (not even sure what the warning should say) and let you do it in January if you don't mind.

Reply:
Issue created here https://github.com/bh2smith/safe-airdrop/issues/326